notifications.yaml

```diff
@@ -0,0 +1,220 @@
+swagger: "2.0"
+info: 
+  version: "0.0.1"
+  title: "Notification API"
+  description: "Notification API."
+host: "localhost"
+basePath: "/api/v1"
+schemes: 
+  - "https"
+paths: 
+  /Messages/{id}: 
+    parameters: 
+      - 
+        name: "id"
+        in: "path"
+        required: true
+        type: "string"
+        format: "uuid"
+    get: 
+      operationId: "getMessage"
+      summary: "Get Message"
+      tags: 
+        - "Messages"
+      description: "Retrieve a message already submitted for delivery."
+      responses: 
+        200: 
+          description: "Message found."
+          schema: 
+            $ref: "#/definitions/Message"
+  /Users/{id}/Messages: 
+    parameters: 
+      - 
+        name: "id"
+        in: "path"
+        required: true
+        type: "string"
+        format: "uuid"
```

Comment:
I'm thinking about this. The fiscal number is discoverable. Probably the use case here is to get messages only for the logged in user (the one that shares his uuid with the service provider).